services/dask-sidecar/docker/boot.sh

```diff
@@ -4,24 +4,30 @@ set -o nounset
 
 IFS=$(printf '\n\t')
 
-INFO="INFO: [$(basename "$0")] "
+GREEN='\033[0;32m'
+NC='\033[0m' # No Color
+
+# Function to print an INFO message in green
+print_info() {
+  echo "${GREEN}INFO [$(basename "$0")]:${NC}$1"
+}
 
 # BOOTING application ---------------------------------------------
-echo "$INFO" "Booting in ${SC_BOOT_MODE} mode ..."
-echo "  User    :$(id "$(whoami)")"
-echo "  Workdir :$(pwd)"
-echo "  env     :$(env)"
+print_info "Booting in ${SC_BOOT_MODE} mode ..."
```

Comment:
we should use this `print_info` in all boot. I wonder if something like https://github.com/just-containers/s6-overlay has tooling for that.